chart/flux/templates/deployment.yaml

```diff
@@ -34,6 +34,10 @@ spec:
       - name: git-keygen
         emptyDir:
           medium: Memory
+      - name: docker-creds   
+        secret:
+          secretName: {{ template "flux.fullname" . }}-docker-creds
```

Comment:
It doesn't look like this secret will be created when the chart is installed; will that mean the deployment doesn't run?

Reply:
Yeah my fail. Not sure how it went in there. Probably i messed around with one version. Fixing